recipes/libnghttp2/all/conanfile.py

```diff
@@ -108,6 +144,10 @@ def generate(self):
         tc.generate()
 
         tc = CMakeDeps(self)
+        if self.options.with_app:
+            tc.set_property("libev", "cmake_file_name", "LIBEV")
```

Comment:
The variables are uppercase, but the CMake file name is not, causing missing package when building libnghttp2:

```
-- Could NOT find Libev (missing: LIBEV_LIBRARY LIBEV_INCLUDE_DIR) (Required is at least version "4.11")
-- Could NOT find Libcares (missing: LIBCARES_LIBRARY LIBCARES_INCLUDE_DIR) (Required is at least version "1.7.5")
-- Could NOT find ZLIB (missing: ZLIB_LIBRARY ZLIB_INCLUDE_DIR) (Required is at least version "1.2.3")
-- Could NOT find Libbrotlienc (missing: LIBBROTLIENC_LIBRARY LIBBROTLIENC_INCLUDE_DIR) (Required is at least version "1.0.9")
-- Could NOT find Libbrotlidec (missing: LIBBROTLIDEC_LIBRARY LIBBROTLIDEC_INCLUDE_DIR) (Required is at least version "1.0.9")
-- Could NOT find OpenSSL, try to set the path to OpenSSL root folder in the system variable OPENSSL_ROOT_DIR (missing: OPENSSL_CRYPTO_LIBRARY OPENSSL_INCLUDE_DIR) (Required is at least version "1.1.1")
-- Could NOT find Libngtcp2 (missing: LIBNGTCP2_LIBRARY LIBNGTCP2_INCLUDE_DIR) (Required is at least version "1.0.0")
-- Could NOT find Libngtcp2_crypto_quictls (missing: LIBNGTCP2_CRYPTO_QUICTLS_LIBRARY LIBNGTCP2_CRYPTO_QUICTLS_INCLUDE_DIR) (Required is at least version "1.0.0")
-- Could NOT find Libnghttp3 (missing: LIBNGHTTP3_LIBRARY LIBNGHTTP3_INCLUDE_DIR) (Required is at least version "1.1.0")
-- Could NOT find Systemd (missing: SYSTEMD_INCLUDE_DIRS SYSTEMD_LIBRARIES) (Required is at least version "209")
-- Could NOT find Jansson (missing: JANSSON_LIBRARY JANSSON_INCLUDE_DIR) (Required is at least version "2.5")
```

https://c3i.jfrog.io/c3i/misc-v2/logs/pr/23979/3-linux-gcc/libnghttp2/1.62.1//2a6d4ded4efa32cdae04eb5160eba3173013407e-build.txt


The project wants Capitalized names (I didn't check what Conan is providing):

- https://github.com/nghttp2/nghttp2/blob/d13a5758373931064636c1641db6277db45552dc/CMakeLists.txt#L61
- https://github.com/nghttp2/nghttp2/blob/d13a5758373931064636c1641db6277db45552dc/CMakeLists.txt#L87